front/app/containers/Admin/projects/components/ProjectRow/index.tsx

```diff
@@ -160,6 +160,7 @@ const ProjectRow = ({
           {!hideMoreActions && (
             <ProjectMoreActionsMenu
               projectId={projectId}
+              folderId={folderId ? folderId : undefined}
               setError={setError}
               setIsRunningAction={handleActionLoading}
```

Comment:
We use `folderId` for checks in this component, but we were never providing it where it's used.